txpipe/blinding.py

```diff
@@ -0,0 +1,226 @@
+###
+#
+# TO BE DEPRECATED VERY SOON
+# Leaving here to be able to see what is going on
```

Comment:
Delete this if it's not used now.

Reply:
Bah, I thought i've deleted that. Should be gone now.